src/foam/comics/v2/userfeedback/UserFeedbackException.js

```diff
@@ -7,7 +7,9 @@
 foam.CLASS({
   package: 'foam.comics.v2.userfeedback',
   name: 'UserFeedbackException',
-  implements: ['foam.core.Exception'],
+  extends: 'foam.core.AbstractException',
+  generateNativeException: true,
```

Comment:
@nick-prat @KernelDeimos is 'native' java when generating java, swift for swift, ... ?

Reply:
@jlhughes I'm not very familiar with swift, but from what I've read it seems that the throwable type is an interface instead of a class, so it renders this whole solution redundant.  It might be a good idea to change this to `generateJavaException` instead, as it's a java specific problem & solution.

Reply:
I agree with @nick-prat that `generateJavaException` makes more sense, but now I'm thinking exceptions should just always do that unless they're excluded from java generation alltogether, right?

Reply:
I think keeping it opt-in is good as I'd rather have the compile load lower then a have a large amount of unused java exception classes.

Reply:
@nick-prat @KernelDeimos re: opt-in - yes, I was referring to what 'native' meant.  So change to generateJavaException is that's all it can do at the moment. and also newNative to javaException() or something like that.

Reply:
@jlhughes Done